adapter/inventory.bzl

```diff
@@ -0,0 +1,27 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+def _inventory_gen(name, packages, out):
+  args = ""
+  for k, v in packages.items():
+    args += "-p %s:%s " % (k, v)
+  
+  native.genrule(
+      name = name+"_gen",
+      outs = [out],
```

Comment:
Naive bazel question, we are not passing the out to the tool, who is filling in the out file ?